test/tickets/LDEV1068.cfc

```diff
@@ -0,0 +1,18 @@
+component extends="org.lucee.cfml.test.LuceeTestCase"{
+	function run( testResults , testBox ) {
+		describe( "list to array function with numeric value", function() {
+			it("simple list to array function", function( currentSpec ){
```

Comment:
The `describe()` and `it()` methods appear to be pasted from another test case. They should probably be rewritten for this test case. Maybe something like:

```cfml
describe( "SerializeJSON", function() {
    it("should return uppercase column names inside DATA structure", function( currentSpec ){
```